spinalcordtoolbox/scripts/sct_compute_snr.py

```diff
@@ -129,7 +130,7 @@ def weighted_std(values, weights):
     return np.sqrt(variance)
 
 
-def main(argv=None):
+def main(argv: Sequence[str]):
```

Comment:
Hmm. It looks like we encountered https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3759 again:

```
 =================================== FAILURES ===================================
__________________ test_sct_compute_snr_single_3d_mask_object __________________
Traceback (most recent call last):
  File "/home/runner/sct_0.0/testing/cli/test_cli_sct_compute_snr.py", line 267, in test_sct_compute_snr_single_3d_mask_object
    assert snr == pytest.approx(np.sqrt(2*SIGNAL_OBJECT**2), rel=0.05)
AssertionError: assert 1512.0527270220502 == 1414.213562373095 ± 7.1e+01
  comparison failed
  Obtained: 1512.0527270220502
  Expected: 1414.213562373095 ± 7.1e+01
=========================== short test summary info ============================
FAILED ../../../sct_0.0/testing/cli/test_cli_sct_compute_snr.py::test_sct_compute_snr_single_3d_mask_object
```

I've retriggered the workflow, and I'm presuming it's going to pass (based on what's written in https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3759), but it's strange to see the same error twice...

Reply:
Thanks for the cross-reference!